src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultHtmlFactsService.cs

```diff
@@ -28,54 +28,78 @@ public override bool TryGetElementInfo(SyntaxNode element, out SyntaxToken conta
             return false;
         }
 
-        public override bool TryGetAttributeInfo(SyntaxNode attribute, out SyntaxToken containingTagNameToken, out string selectedAttributeName, out SyntaxList<RazorSyntaxNode> attributeNodes)
+        public override bool TryGetAttributeInfo(
+            SyntaxNode attribute,
+            out SyntaxToken containingTagNameToken,
+            out TextSpan? prefixLocation,
```

Comment:
nullable because this method returns `true` in the case of a miscellaneous attribute in which we need to notify consumers that there's no corresponding location(s)